state/indexer/block/kv/kv.go

```diff
@@ -82,8 +91,101 @@ func (idx *BlockerIndexer) Index(bh types.EventDataNewBlockEvents) error {
 	if err := idx.indexEvents(batch, bh.Events, height); err != nil {
 		return fmt.Errorf("failed to index FinalizeBlock events: %w", err)
 	}
+	err = batch.WriteSync()
+	return err
+}
+
+func getKeys(indexer BlockerIndexer) [][]byte {
```

Comment:
is this used only in `kv_test.go` ? If so, I think it should be moved there.

Reply:
It's also exported for testing in `state/indexer/block/kv/export_testing.go`